services/vcpkg/vcpkg-version.tester.js

```diff
@@ -1,11 +1,23 @@
-import { isSemver } from '../test-validators.js'
+import Joi from 'joi'
 import { createServiceTester } from '../tester.js'
 
 export const t = await createServiceTester()
 
-t.create('gets the port version of entt')
+t.create('gets the port version of entt using `version` field')
```

Comment:
**note**: These tests aim to validate the 4 different fields that can be included in Vcpkg manifests. However, if these specific ports use suddenly a different field, the tests won't cover all of them anymore. If this is a problem, we could imagine having a different collection of tests against static data.